.github/settings.yml

```diff
@@ -68,8 +68,6 @@ labels:
   - name: time-sensitive
     description: issues or PRs that have a due date
     color: b60205
-  - name: getting started
-    color: 3B8924
```

Comment:
Not used.
We have good-first-issue and help-wanted that could replace this.

Reply:
I think this is used in the discussions board (see the Q&A section), but I'm not sure it's well used or helpful.

Reply:
Maybe there is a better label for these kinds of questions/discussions?

Reply:
Hm, never thought about the discussions. Thanks for pointing that out.

I will get "getting started" back in the labels list.

We can decide later what label can be more suitable and work it in a separate PR..

Reply:
Done